### PR TITLE
chore: Update CHANGELOG to release a bug fix version

### PR DESCRIPTION
**Pull request by karasusan**
- Update README
- Add changes of the version `2.3.2` to CHANGELOG
- Upgrade Unity editor version `2019.4.19.f1`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ If you are interested in the streaming solution with WebRTC, you can check [Unit
 
 ## Documentation
 
-- [English](https://docs.unity3d.com/Packages/com.unity.webrtc@2.1/manual/index.html)
-- [Japanese](https://docs.unity3d.com/Packages/com.unity.webrtc@2.1/manual/jp/index.html)
+- [English](https://docs.unity3d.com/Packages/com.unity.webrtc@2.3/manual/index.html)
+- [Japanese](https://docs.unity3d.com/Packages/com.unity.webrtc@2.3/manual/jp/index.html)
```
Comment (kannan-xiao4) [inline README.md:13]: maybe, we can use `@latest` url.  below url redirect latest version url.
```
https://docs.unity3d.com/Packages/com.unity.webrtc@latest/index.html
```

and, Does Japanese document exist? this Japanese url is invalid.

**Review by kannan-xiao4 — APPROVED** [review]
basically, lgtm.

File: README.md (hunk 1 above)
Comment (karasusan) [reply README.md:13]: Sure. I will fix it below
https://docs.unity3d.com/ja/Packages/com.unity.webrtc@latest/index.html
https://docs.unity3d.com/Packages/com.unity.webrtc@latest/index.html

Comment (karasusan) [reply README.md]: Fixed it

**Pull request merged**